apps/src/lib/script-editor/ScriptEditor.jsx

```diff
@@ -155,6 +162,39 @@ const ScriptEditor = React.createClass({
             style={styles.input}
           />
         </label>
+        <h3>Project widget options</h3>
+        <label>
+          Project widget visible
+          <input
+            name="project_widget_visible"
+            type="checkbox"
+            defaultChecked={this.props.projectWidgetVisible}
+            style={styles.checkbox}
+          />
+          <p>
+            If checked this script will have the projects widget (recent projects and new
+            project buttons) visible in stage extras.
+          </p>
+        </label>
+        <label>
+          Project widget new project types
+          <p>
+            Select up to 4 project type options to appear in the 'Start a new project' section. Select
+            <a className="select_none"> none </a>
```

Comment:
That makes way more sense - I ended up copying this from another place that has a multi-select and should've converted it a bit more.